cmake/modules/FindCocoa.cmake

```diff
@@ -0,0 +1,25 @@
+#  -------------------------------------------------------------------------
+#  Copyright (C) 2018 BMW Car IT GmbH
+#  -------------------------------------------------------------------------
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#  -------------------------------------------------------------------------
+
+IF(CMAKE_SYSTEM_NAME MATCHES "Darwin")
+
+    FIND_PATH(Cocoa_INCLUDE_DIRS Cocoa.h)
```

Comment:
Since this is only required for getting a macOS renderer to work and this is outside of the scope of this pull request, I removed the entire `FindCocoa.cmake` script.